DOL.WHD.Section14c.Test/Business/Validators/SignatureValidatorTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using DOL.WHD.Section14c.Business.Validators;
+using FluentValidation.TestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace DOL.WHD.Section14c.Test.Business.Validators
+{
+    [TestClass]
+    public class SignatureValidatorTests
+    {
+        private ISignatureValidator _signatureValidator;
+        public SignatureValidatorTests()
+        {
+            _signatureValidator = new SignatureValidator();
+        }
+
+        [TestMethod]
+        public void Should_Require_Agreement()
+        {
+            _signatureValidator.ShouldHaveValidationErrorFor(x => x.Agreement, null as bool?);
```

Comment:
Is there value in also testing `false`?  I know the result should be the same so this is totally acceptable - just curious.

Also, please add tests to verify the cases where there aren't validation errors.